snaps_tests/demos_tests/test_git.py

```diff
@@ -0,0 +1,36 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2015, 2016, 2017 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import platform
+
+import snaps_tests
+
+
+class GitTestCase(snaps_tests.SnapsTestCase):
+
+    snap_content_dir = 'git'
+
+    def test_gopaste(self):
+        if platform.machine() == 'armv7l':
```

Comment:
please use an instance of `project_options` for these so we don't have to make a global change everywhere later.